app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.posts
+  end
```

Comment:
| @Alibaba2023 User Show | Microverse User Show |
|-|-|
| ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/d0f24099-0aef-44d5-967d-d2dfb5d82e2a) | ![image](https://github.com/Alibaba2023/blog_app/assets/93039359/f638e6cd-e032-4675-8841-3f1450be9e59) |

- [ ] It would be nice if you worked on the User Show page to follow the project requirements 👍 
- [ ] It would be great if you use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to follow the project requirements 😊